governance/polls/Proposal - Prioritising RWA - September 21, 2020.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Could you maybe expand to 'Real World Asset Collateral' in the filename? I'm worried someone might mistake it for an asset ticker. Not a major issue though.